tests/find/test-suite-1/test.exists.js

```diff
@@ -41,7 +40,6 @@ testCases.push(function (dbType, context) {
       ]).then(function () {
         return db.find({
           selector: {
-            _id: { $gt: null},
```

Comment:
Nice clean up. Could you add another test that runs the explain method of query.  An `exists: false` and `$exists: true` selector on its own should not use an index but always fallback to all_docs.